pitest-maven-verification/src/test/java/org/pitest/PitMojoIT.java

```diff
@@ -149,13 +157,15 @@ public void shouldCorrectlyTargetTestsWhenMultipleBlocksIncludeALine() throws Ex
    */
   @Test
   public void shouldSkipSiteReportGeneration() throws Exception {
-    File testDir = prepare("/pit-site-skip");
-
-    verifier.executeGoals(Arrays.asList("test", "org.pitest:pitest-maven:mutationCoverage", "site"));
-
+    File testDir = prepareSiteTest("/pit-site-skip");
```

Comment:
is the comment of this test really needed? I think the name is perfect and the comment redundant.